samigo/samigo-app/src/webapp/jsf/samlite/samLiteEntry.jsp

```diff
@@ -212,6 +232,40 @@
 		</h:form>
 		</div>
  		<!-- end content -->
+		<script>
+			function pre_show_editor() {
+				show_editor('samLiteEntryForm:data', document.getElementById('samLiteEntryForm:data_textinput_current_status').value, 1000000);
+			}
+			function chef_setupformattedtextarea(client_id, shouldToggle, frame_id, max_chars) {
+				if (shouldToggle == true) {
+					var input_text = document.getElementById(client_id);
+					var input_text_value = input_text.value;
+					var input_text_encoded = encodeHTML(input_text_value);
+					input_text.value = input_text_encoded;
+				}
+
+				config = ''
+				if (max_chars) {
+					config = 
+						{
+							wordcount: {'maxCharCount' : max_chars},
+							toolbarCanCollapse: false,
+							toolbar: 'Basic',
+							toolbar_Basic: [
+								['Source','-','Bold','Italic','Underline','-','TextColor','BGColor','-','Subscript','Superscript','-','Image','Link','Unlink'],
+							],
+						}
+				}
+				
+				sakai.editor.launch(client_id, config,'450','240');
+			}
+		</script>
+		<h:panelGroup styleClass="" rendered="#{samLiteBean.richTextarea}">
```

Comment:
h:panelGroup will render a \<span\>. It is possible to use h:outputScript or something similar??